### ISSUE TITLE
Regarding input normalization and loss profile during training.

### ISSUE THREAD

**Issue by aditya-vora** [svishwa/crowdcount-mcnn#16]
Is there any sort normalization of the input done during the training i.e. mean subtraction or centralization of the data. I see that you have implemented batch normalization layer in the network which helps to get rid of normalization. However, I also saw that you have kept it off during training. 

Also, how was the loss profile when you trained the network. Was it very less i.e. in the orders of 10^-3 and so on?

**Comment by svishwa** [comment]
I did not use any normalization on input data. Though batch normalization is implemented, I did not use it since the original paper does not use it. Also, the training images are of different sizes, so it will be tricky to use batch norm. 

I do not remember the exact loss profile since it has been a while. It was low as you mentioned. If your next question is why so low error - it is because 1. it is per pixel error and 2. density map pixel values are very low.
